frontend/pages/hosts/details/cards/Certificates/CertificatesTable/CertificatesTable.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import { Row } from "react-table";
+
+import { IHostCertificate } from "interfaces/certificates";
+
+import TableContainer from "components/TableContainer";
+import CustomLink from "components/CustomLink";
+
+import generateTableConfig from "./CertificatesTableConfig";
+
+const baseClass = "certificates-table";
+
+interface ICertificatesTableProps {
+  data: IHostCertificate[];
+  showHelpText: boolean;
+  onSelectCertificate: (certificate: IHostCertificate) => void;
+}
+
+const CertificatesTable = ({
+  data,
+  showHelpText,
+  onSelectCertificate,
+}: ICertificatesTableProps) => {
+  const tableConfig = generateTableConfig();
+
+  const onClickTableRow = (row: Row<IHostCertificate>) => {
+    onSelectCertificate(row.original);
+  };
+
+  const helpText = showHelpText ? (
+    <p>
+      Showing certificates in the system keychain. To get all certificates, you
+      can query the certificates table.{" "}
```

Comment:
The second sentence is not applicable to iOS/iPadOS hosts. Should we hide it for other platforms as well as from device users (who can't run queries)?

Reply:
This text is only ever showed on the host details page for darwin hosts. `showHelpText` is passed in from `CertificatesCard`